### PR TITLE
Tag uploaded blobs on Azure and fix tags in cli tests

### PR DESCRIPTION
**Pull request by pdostal**
- Related ticket: [poo#126914](https://progress.opensuse.org/issues/126914)
- Verification run: [existing](https://pdostal-server.suse.cz/t4283), [new](https://pdostal-server.suse.cz/t4277), [cli](https://pdostal-server.suse.cz/t4287)

### CODE REVIEW

File: lib/publiccloud/azure.pm [hunk 1]
```diff
@@ -143,6 +144,11 @@ sub upload_img {
 
     my $arch = (check_var('PUBLIC_CLOUD_ARCH', 'arm64')) ? 'Arm64' : 'x64';
 
+    my $job_id = get_current_job_id();
+    my $openqa_url = get_var('OPENQA_URL');
```
Comment (ricardobranco777) [inline lib/publiccloud/azure.pm:148]:
```suggestion
    my $openqa_url = get_required_var('OPENQA_URL');
```

File: tests/publiccloud/aws_cli.pm [hunk 2]
```diff
@@ -42,7 +42,8 @@ sub run {
     my $machine_name = "openqa-cli-test-vm-$job_id";
     my $security_group_name = "openqa-cli-test-sg-$job_id";
     my $openqa_ttl = get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300);
-    my $created_by = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
+    my $openqa_url = get_var('OPENQA_URL');
```
Comment (ricardobranco777) [inline tests/publiccloud/aws_cli.pm:45]:
```suggestion
    my $openqa_url = get_required_var('OPENQA_URL');
```

File: tests/publiccloud/azure_cli.pm [hunk 3]
```diff
@@ -39,7 +39,8 @@ sub run {
     my $machine_name = "openqa-cli-test-vm-$job_id";
 
     my $openqa_ttl = get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300);
-    my $created_by = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
+    my $openqa_url = get_var('OPENQA_URL');
```
Comment (ricardobranco777) [inline tests/publiccloud/azure_cli.pm:42]:
```suggestion
    my $openqa_url = get_required_var('OPENQA_URL');
```

File: tests/publiccloud/google_cli.pm [hunk 4]
```diff
@@ -46,7 +46,8 @@ sub run {
 
     my $machine_name = "openqa-cli-test-vm-$job_id";
     my $openqa_ttl = get_var('MAX_JOB_TIME', 7200) + get_var('PUBLIC_CLOUD_TTL_OFFSET', 300);
-    my $created_by = get_var('PUBLIC_CLOUD_RESOURCE_NAME', 'openqa-vm');
+    my $openqa_url = get_var('OPENQA_URL');
```
Comment (ricardobranco777) [inline tests/publiccloud/google_cli.pm:49]:
```suggestion
    my $openqa_url = get_required_var('OPENQA_URL');
```

**Review by ricardobranco777 — APPROVED** [review]
Small suggestion. Otherwise LGTM.

**Review by ilausuch — APPROVED** [review]

**Pull request merged**